daemon/service.go

```diff
@@ -453,3 +516,9 @@ func (s *ControlService) DriverStates() ([]*protocol.DriverImageState, error) {
 
 	return out, nil
 }
+
+func killDriver(pool *DriverPool, drv Driver, err error) {
```

Comment:
A method on pool? Or maybe an additional parameter to existing `killDriver` method on `pool`?